h2/src/main/org/h2/value/Value.java

```diff
@@ -702,17 +702,73 @@ public final Value convertTo(int targetType, CastDataProvider provider) {
     }
 
     /**
-     * Convert a value to the specified type.
+     * Convert a value to the specified type without taking scale and precision
+     * into account.
      *
-     * @param targetType the type of the returned value
-     * @param provider the cast information provider
-     * @param column the column (if any), used for to improve the error message if conversion fails
+     * @param targetType
+     *            the type of the returned value
+     * @param provider
+     *            the cast information provider
+     * @return the converted value
+     */
+    public final Value convertTo(TypeInfo targetType, CastDataProvider provider) {
+        return convertTo(targetType.getValueType(), targetType.getExtTypeInfo(), provider, null);
+    }
+
+    /**
+     * Convert a value to the specified type without taking scale and precision
+     * into account.
+     *
+     * @param targetType
+     *            the type of the returned value
+     * @param provider
+     *            the cast information provider
+     * @param column
+     *            the column, used for to improve the error message if
```

Comment:
for to -> to

Reply:
4e615c59563fc19d20080554d41cd5e94526712e